dispatch/apps/events/models.py

```diff
@@ -57,3 +67,40 @@ def save_image_from_url(self, url):
         )
 
         self.save()
+
+
+@receiver(post_update, sender=Event)
+def send_approved_email(sender, instance, **kwargs):
+
+    if not instance.is_approved_email:
```

Comment:
Shouldn't the check include the `is_submission` field?

I think it's possible to update an event without approving it, which would trigger this function and send out an email. Unless I'm missing something?